### PR TITLE
dev-python/*: Port to python3.10

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: dev-python/sniffio/sniffio-1.2.0.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
 inherit distutils-r1
 
 DESCRIPTION="Sniff out which async library your code is running under"
```
Comment (mgorny) [inline dev-python/sniffio/sniffio-1.2.0.ebuild:9]: Convert to `epytest`.

**Review by mgorny — CHANGES_REQUESTED** [review]

File: dev-python/sniffio/sniffio-1.2.0.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-python/sniffio/sniffio-1.2.0.ebuild:9]: done

File: dev-python/fastimport/fastimport-0.9.12.ebuild [hunk 2]
```diff
@@ -13,7 +13,6 @@ SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.tar.gz"
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS=""
```
Comment (mgorny) [inline dev-python/fastimport/fastimport-0.9.12.ebuild:16]: Did you actually test this version?

Comment (jsmolic) [reply dev-python/fastimport/fastimport-0.9.12.ebuild:16]: Yeah, tests passed on my end successfully

**Comment by mgorny** [comment]
Merging all but fastimport.

File: dev-python/fastimport/fastimport-0.9.12.ebuild (hunk 2 above)
Comment (mgorny) [reply dev-python/fastimport/fastimport-0.9.12.ebuild:16]: But you do realize that this version has no keywords?

Comment (jsmolic) [reply dev-python/fastimport/fastimport-0.9.12.ebuild:16]: Yup, I was just about to add 3.10 to 0.9.11 as well

**Pull request closed without merging**